dbcon/mysql/columnstore.cnf

```diff
@@ -1,6 +1,4 @@
 [mysqld]
-lower_case_table_names=1
```

Comment:
Removed this per request in MDEV-22197.

Reply:
Will this affect our regression tests at all?

Reply:
No. We don't have queries against objects with names in different case

Reply:
I'll ask Roman to set the option for our CI b/c it takes too much time to unwind all the tables names in mixed case style.